playwright-tests/tests/proposals.spec.js

```diff
@@ -194,34 +189,12 @@ test.describe("Wallet is connected", () => {
       .locator(".CodeMirror textarea");
     await descriptionArea.focus();
     await descriptionArea.pressSequentially(
-      `The test proposal description. And mentioning @petersal`,
-      {
-        delay: delay_milliseconds_between_keypress_when_typing,
-      }
+      "The test proposal description. And mentioning @petersalomonsen.near. Also mentioning @megha19.near"
     );
 
     await pauseIfVideoRecording(page);
 
-    await page.frameLocator("iframe").getByText("petersalomonsen.near").click();
-
```

Comment:
@Megha-Dev-19 you may want to keep this, because this is testing some properties of the markdown editor, This was the fix we made for pressing backspace leaving only the @, and then start typing again ( when the dropdown used to not show up again ). We may want to have such tests